### PR TITLE
2787 Updated the dependencies to the latest

### LINKED ISSUE #116: Update dependencies and resources
#### Describe the desired behavior
Use Angular 13, React 17, MUI v5 and update other dependencies used by the CLI

#### Describe the current behavior
These dependencies are out of date

#### Is this request related to a current issue?
No

#### Additional Context

Some of these dependencies may already be handled by updates to the templates themselves, but we should check to see what else there is that is specified in the CLI and need to be updated.

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes #116 

#### Changes proposed in this Pull Request:
- Updated dependencies to the latest

#### To Test:
- create a new blui project using cli :
npx -p @brightlayer-ui/cli blui new <framework>
- start the new project

### CODE REVIEW

**Comment by jeffvg** [comment]
@bkarambe your devDependencies pretty much match what I have in my branch, so I'll close it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,12 +4,13 @@
 
 ### Changed
 
-- Update angular projects to use angular 13.
-- Updated bundle size budgets when building angular projects.
+-   Updated dependencies to the latest (#116)[https://github.com/brightlayer-ui/brightlayer-ui-cli/issues/116].
```
Comment (daileytj) [inline CHANGELOG.md:7]: I'm about to push a commit to fix this, but this markdown link syntax is wrong so it ends up showing the "#116" twice. I would recommend adding a markdown previewer extension for vscode so you can see what these will look like.

**Comment by daileytj** [comment]
Do we want to see what it would take to update react-native to the latest version? We are locked down on 0.64.1, but RN is currently on 0.67.4?

**Review by jeffvg — APPROVED** [review]

**Review by daileytj — APPROVED** [review]
I'm having issues getting RN 0.66+ to build properly so let's get this in and I'll add a story to get RN dependencies updated to latest

**Pull request merged**